toolkit/resources/dracutmodules/90mountesppartition/10-mountesppartition.conf

```diff
@@ -0,0 +1 @@
+add_dracutmodules+=" mountesppartition "
```

Comment:
There is also the [toolkit/tools/internal/resources](https://github.com/microsoft/azure-linux-image-tools/tree/main/toolkit/tools/internal/resources) directory, which embeds the files into the binary.